app/assets/javascripts/application.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+document.onreadystatechange = function () {
```

Comment:
Do we need to think about a way to have this javascript initialise only when this form is available? as it stands now I think it'll run on every page.